jest.config.js

```diff
@@ -79,6 +79,7 @@ module.exports = {
   moduleNameMapper: {
     // TODO: improve: jest will not work with 'module-alias', so we have define the alias here again!
     // see https://github.com/ilearnio/module-alias/issues/46
+    '^@lib/utils$': '<rootDir>/platform/lib/utils/index.js',
```

Comment:
Just wondering: wouldn't it help to just add `require('module-alias/register');` at the top of this file? My guess would be that this file is the entry point for jest and all other files are only evaluated after.

Reply:
@matthiasrohmer This does not work. I also tries to import the module-alias in a globalSetup module without success.